lib/azure_fence.py.py

```diff
@@ -350,39 +412,43 @@ def get_azure_compute_client(config):
     cloud_environment = get_azure_cloud_environment(config)
     credentials = get_azure_credentials(config)
 
-    if cloud_environment:
-        if (config.Cloud.lower() == "stack") and not config.MetadataEndpoint:
-                fail_usage("metadata-endpoint not specified")
+    from azure.mgmt.compute import __version__

```

Comment:
__version__ isnt available in azure_mgmt_compute-27.2.0:
```
Traceback (most recent call last):
  File "./fence_azure_arm", line 272, in <module>
    main()
  File "./fence_azure_arm", line 250, in main
    compute_client = azure_fence.get_azure_compute_client(config)
  File "/usr/share/fence/azure_fence.py", line 415, in get_azure_compute_client
    from azure.mgmt.compute import __version__
ImportError: cannot import name '__version__'
```

Reply:
Mhh ok than I tested it with the wrong version.
Do you have a list with the dependency versions that you are using when you run the fence_agent on RHEL8?

Reply:
azure-common-1.1.28
azure-core-1.24.2
azure-mgmt-compute-27.2.0
azure-mgmt-core-1.3.2
azure-mgmt-network-20.0.0
azure-identity-1.10.0
msrest-0.7.1
msal-1.27.0
msal-extensions-1.0.0